lib/.gitignore

```diff
@@ -7,3 +7,11 @@
 
 npm-debug.log
 junit.xml
+
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
We have a top-level `.gitignore` which woul be the right place for these entries that are in both `lib` and `gallery`